react/components/ExtensionPortal.tsx

```diff
@@ -0,0 +1,29 @@
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import {createPortal as reactCreatePortal} from 'react-dom'
```

Comment:
Why the rename?

Reply:
I noticed now that it was indeed unnecessary, removed on 9263185